app/assets/javascripts/templates/story.ejs

```diff
@@ -49,7 +49,7 @@
       <% }); %>
     </span>
   <% } %>
-  <%= story.escape('title') %>
+  <%- story.escape('title') %>
```

Comment:
you removed the "=" (equal) for a "-" ?? But this will just make it disappear. <%- will still print it?